spec/install/gems/flex_spec.rb

```diff
@@ -200,11 +200,11 @@
             rack (= 0.9.1)
 
           In Gemfile:
-            rack_middleware (>= 0) ruby depends on
-              rack (= 0.9.1) ruby
+            rack-obama (= 2.0), resolved to 2.0, depends on
+              rack (= 1.2)
```

Comment:
Now that I'm reading it, I'd prefer `rack-obama (= 2.0) was resolved to 2.0, which depends on\n  rack(= 1.2)`. Sorry to nitpick on you. :stuck_out_tongue_closed_eyes: